src/NSwag.CodeGeneration.TypeScript/Models/TypeScriptOperationModel.cs

```diff
@@ -36,13 +36,13 @@ public class TypeScriptOperationModel : OperationModelBase<TypeScriptParameterMo
             _settings = settings;
             _generator = generator;
 
-            var parameters = GetActualParameters();
+            var parameters = GetActualParameters().OrderBy(p => p.Position).ToList();
```

Comment:
I think the `ToList()` calls should be removed from here and below, there's no need to materialize as list as it is done at the end anyway.